unicodetools/data/ucd/dev/ArabicShaping.txt

```diff
@@ -847,9 +847,28 @@ A873; PHAGS-PA CANDRABINDU; U; No_Joining_Group
 
 # Arabic Extended-C Characters
 
-10EC2; DAL WITH VERTICAL 2 DOTS BELOW; R; DAL
```

Comment:
Why are these lines removed?

Reply:
Pilot error, when I picked up Manish’s correction (as noted in the SAH issue the proposed lines were wrong initially) I pasted over the whole block.